configurator/src/main.rs

```diff
@@ -274,15 +281,22 @@ fn main() -> Result<(), anyhow::Error> {
         std::thread::sleep(std::time::Duration::from_secs(1));
     }
     let config: Config = serde_yaml::from_reader(File::open("/root/.lnd/start9/config.yaml")?)?;
+    let alias = get_alias(&config)?;
+    let control_tor_address: String = config.control_tor_address;
+    let watchtower_tor_address: String = config.watchtower_tor_address;
+    let peer_tor_address: String = config.peer_tor_address;
```

Comment:
Do you need these type annotations?

Reply:
Nope. I made no changes to this code. Shall I remove?

Reply:
Removed